README.md

```diff
@@ -17,6 +17,20 @@ mkdir -p $GOPATH/src/pebble-dev
 git clone https://github.com/pebble-dev/rebblestore-api.git $GOPATH/src/pebble-dev/rebblestore-api
 ```
 
+Then, you will need to generate a local (self signed) TLS certificate. The server only supports HTTPS, for obvious security reasons.
+```shell
+cd $GOPATH/src/pebble-dev
```

Comment:
Shouldn't this be `cd $GOPATH/src/pebble-dev/rebblestore-api`?

Reply:
Yes, fixing it now